src/components/ApplicationTile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import Text from '../Text';
+import styles from './styles.css';
+import getFormattedTimeDifference from '~/utils/getFormattedTimeDifference';
+
+interface Props {
+  companyName: string;
+  roleName: string;
+  status: 'sent' | 'inProgress' | 'rejected' | 'accepted' | 'hired' | 'archived';
+  salary: {
+    from: number;
+    to: number;
+  };
+  linkToOffer: string;
+  createdAt: Date;
+  className?: string;
+}
+
+const ApplicationTile = ({
+  companyName,
+  roleName,
+  status,
+  salary,
+  linkToOffer,
+  createdAt,
+  className,
+}: Props) => {
+  const { t } = useTranslation('applications');
+
+  return (
+    <div className={classNames(styles.root, className)}>
+      <aside className={classNames(styles.aside, styles[status])} />
+      <main className={styles.main}>
+        <div className={styles.row}>
+          <Text variant='medium'>{companyName}</Text>
+
+          <span className={styles.statusWrapper}>
```

Comment:
maybe this could be another component like `ApplicationTileStatus` ?

Reply:
Done. Also moved classes with only color to inline style (as it requires less code and is easier to read)